tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -3277,9 +3297,9 @@ void test_remove() {
     EXPECT(!exists(dirname, ec));
     EXPECT(good(ec));
 
-    EXPECT(throws_filesystem_error([] { remove(badPath); }, "remove", badPath));
+    remove(badPath); // we ignore bogus invalid paths

```

Comment:
Does "bogus invalid path" mean something different from "invalid path"? This seems redundant.